.github/workflows/vue_deploy.yml

```diff
@@ -5,22 +5,22 @@ on:
   pull_request:
     types: [closed]
     branches:
-      - dev
+      - main
 
 jobs:
   aws-deploy:
     runs-on: ubuntu-latest
     #For google auth
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         # may need to change to version 3
         with:
           ref: dev
```

Comment:
this line here says that you're going to checkout the `dev` branch (on each pull request merged to `main`) and do a build and deployment

Reply:
Thanks for the catch! I though I did that.